examples/OpenTelemetry/Exporter/ConsoleExporter/Program.cs

```diff
@@ -59,7 +59,7 @@ private static async Task MainAsync(string[] args)
                 {
                     try
                     {
-                        var x = await session.ExecuteAsync(new SimpleStatement("SELECT * FROM system.local"));
+                        var x = await session.ExecuteAsync(new SimpleStatement("SELECT * FROM system.local WHERE key='local'"));
```

Comment:
Now that I see that for some reason we do this query every 5 seconds(!), this is a good change!